ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/stack/StackComponentView.kt

```diff
@@ -479,6 +493,9 @@ private fun MainStackComponent(
                         horizontalAlignment = dimension.alignment.toHorizontalAlignmentOrNull(),
                         verticalAlignment = dimension.alignment.toVerticalAlignmentOrNull(),
                     )
+                    .applyIfNotNull(scrollState, stackState.scrollOrientation) { state, orientation ->
+                        scrollable(state, orientation)
```

Comment:
MIght not want to support this for Zstacks... Thought of adding it just in case, but honestly, this would only be needed in cases of adding a nested stack that goes out of the bounds I guess...